### DISCUSSION TITLE
Require OSSF Scorecard

### DISCUSSION THREAD

**Discussion by nikolaydubina** [avelino/awesome-go#5255] [Ideas]
https://securityscorecards.dev

I found it very useful. In fact just by fixing my Go projects to get high score in OSSF Scorecard, I already:
1. fixed CI tests running in Pull Requests
2. hardened security
3. enabled dependabot and GraphQL
4. updated dependencies
5. ... which fixed some obscure [bug](https://github.com/nikolaydubina/smrcptr/issues/4#issuecomment-2012244808) user reported.

<img width="749" alt="image" src="https://github.com/avelino/awesome-go/assets/2933061/b16e604f-564c-4a24-951d-33062efc1ebb">

wow!

OSSF Scorecard is integrated already:
1. pkg.go.dev links to Open Source Insights, which shows OSSF Scorecard
2. it has badge

it would be simple to verify if repo has badge and is registered in OSSF Scorecards.

we may also in future to enforce minimum score (say 5) for projects.

<img width="568" alt="image" src="https://github.com/avelino/awesome-go/assets/2933061/2c55f86a-acb3-4fa5-9c5b-71e40c62c558">
<img width="199" alt="image" src="https://github.com/avelino/awesome-go/assets/2933061/2bfa759d-0fbb-48ca-aee7-97d96d72a954">
<img width="162" alt="image" src="https://github.com/avelino/awesome-go/assets/2933061/fae9d7c2-c96f-4363-8a43-c0a7daa4e364">

I think it is very useful and should be standard for all projects that claim to be awesome-go

(there is literally no excuse to fixing CI and harden security... or at least try)

**Comment by nikolaydubina** [comment]
cc: @avelino

**Reply by avelino** [reply]
are you proposing that you help us correct the ci?

**Reply by nikolaydubina** [reply]
do you mean adding check that new PRs projects need to have OSSF as part of ci here? I think I can add that.

or there is something else is going on with ci here?
